.claude/hooks/poll-pr-status.sh

```
@@ -0,0 +1,321 @@
+if [[ -z "$PR_JSON" || "$PR_JSON" == "" ]]; then
+  # No PR for this branch — nothing to monitor
+  exit 0
+fi
+
+PR_NUM="$(echo "$PR_JSON" | python3 -c "
+import sys, json
+d = json.load(sys.stdin)
+print(d.get('number', ''))
+" 2>/dev/null || echo "")"
+
+PR_URL="$(echo "$PR_JSON" | python3 -c "
```

Shellcheck (0.11.0)

[warning] PR_URL appears unused. Verify use (or export if used externally).

(SC2034)